offload/plugins-nextgen/amdgpu/src/rtl.cpp

```diff
@@ -3476,3 +3472,9 @@ void *AMDGPUDeviceTy::allocate(size_t Size, void *, TargetAllocTy Kind) {
 } // namespace target
 } // namespace omp
 } // namespace llvm
+
+extern "C" {
+llvm::omp::target::plugin::GenericPluginTy *createPlugin_amdgpu() {
```

Comment:
These should use the macro to create the name, one place to change.

Reply:
I tried that, but couldn't figure out how to set a name from a macro within a macro.